src/eureka/S3_data_reduction/source_pos.py

```diff
@@ -24,25 +27,42 @@ def source_pos(data, meta, m, header=False):
     -------
     src_ypos : int
         The central position of the star.
+    src_ypos_exact : float
+        The exact (not rounded) central position of the star.
+    src_ypos_width : float
+        If gaussian fit, the std of the Gaussian fitted to the image
+        Otherwise, array of zeros.
+        
+    Notes
+    -----
+    History:
+    
+    - 2022-07-11 Caroline Piaulet
+        Enable recording of the width if the source is fitted with a Gaussian
+        + add an option to fit any integration (not hardcoded to be the first)
     '''
     if header:
         src_ypos = data.attrs['shdr']['SRCYPOS'] - meta.ywindow[0]
     elif meta.src_pos_type == 'weighted':
         # find the source location using a flux-weighted mean approach
-        src_ypos = source_pos_FWM(data.flux.values, meta, m)
+        src_ypos = source_pos_FWM(data.flux.values, meta, m, integ=integ)
```

Comment:
A PSF width can also be estimated using a FWM-based method. As an example, see SPCA's centroid_FWM function in Photometry_Aperture.py [link](https://github.com/lisadang27/SPCA/blob/30885e1443ce8957c6280efb5846be8caad6a9b2/SPCA/Photometry_Aperture.py#L63) where xw and yw are estimates for the PSF width for Spitzer's photometric data

Reply:
I am not sure what this means here for my PR, although I know the method - can you clarify what this means in terms of modifications?

Reply:
Let's do this in a future PR, but I just meant that it would be nice to be able to get a src_ywidth returned out of this function if one was to use the FWM method instead of Gaussian.